addons/mail/models/ir_actions_server.py

```diff
@@ -109,54 +109,15 @@ class IrActionsServer(models.Model):
         compute='_compute_activity_user_info', readonly=False, store=True)
 
     def _name_depends(self):
-        return super()._name_depends() + [
-            "template_id",
-            "partner_ids",
-            "activity_summary",
-            "activity_type_id",
-            "followers_type",
-            "followers_partner_field_name",
-        ]
+        return [*super()._name_depends(), "template_id", "activity_type_id"]
```

Comment:
I think
```
deps = super()._name_depends()
deps.append(......)
return deps
```

does the trick, doesn't it ?
It is a bit more verbose, but is more efficient (of course in our case with like ~10 elements, it doesn't really matter)